hive_agent/llms/ollama.py

```diff
@@ -4,5 +4,32 @@
 from hive_agent.llms.llms import LLMs
 
 
-class OllamaLLM(LLMs):
-    pass
+from llama_index.core.settings import Settings
+from llama_index.core.agent import ReActAgentWorker
+
+
+class OllamaLLM:
```

Comment:
i'm thinking this should still inherit from the `LLMs` class and then you just override the constructor and create your new agent instance

Reply:
ok sounds good